gdcdictionary/schemas/mass_cytometry_assay.yaml

```diff
@@ -44,7 +53,7 @@ required:
   - assay_method
   - assay_instrument
   - assay_instrument_model
-  - analytes 
+  - analytes
```

Comment:
Should this no longer be required now that you can have a link to either core metadata collections or analytes?